Controllers/TabBarViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TabBarViewController.swift
+//  GoingDutch
+//
+//  Created by Safa özcan on 24.08.2022.
+//
+
+import UIKit
+
+class  TabBarViewController: UITabBarController {
+// MARK: - UI Elements
+    
+// MARK: - Properties
+    
+// MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTabBar()
+    }
+}
+extension TabBarViewController {
+// MARK: - Functions
+    private func setupTabBar() {
+        self.selectedIndex = 0
+        tabBarController?.tabBar.tintColor = UIColor(red: 0.53, green: 0.73, blue: 0.78, alpha: 1.00)
+        self.viewControllers?[0].tabBarItem.image = UIImage(systemName: "house")
+        self.viewControllers?[2].tabBarItem.image = UIImage(systemName: "person")
+        setupMiddleButton()
+    }
+
+    private func setupMiddleButton() {
+        let middleButton = UIButton(frame: CGRect(x: (self.view.bounds.width / 2) - 30, y: -13, width: 60, height: 60))
+        middleButton.setBackgroundImage(UIImage(named: "PlusButtonIcon"), for: .normal)
+        middleButton.layer.shadowColor = UIColor.black.cgColor
+        middleButton.layer.shadowOpacity = 0.1
+        middleButton.layer.shadowOffset = CGSize(width: 6, height: 6)
+        self.tabBar.addSubview(middleButton)
+        middleButton.addTarget(self, action: #selector(plusButtonAction), for: .touchUpInside)
+        self.view.layoutIfNeeded()
+    }
+// MARK: - Actions
+    @objc private func plusButtonAction(sender: UIButton) {
```

Comment:
```suggestion
    @objc
    private func plusButtonAction(sender: UIButton) {
```
It is more readable when written like this.